hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/DiskBalancerService.java

```diff
@@ -505,7 +507,50 @@ private void postCall() {
 
   public DiskBalancerInfo getDiskBalancerInfo() {
     return new DiskBalancerInfo(shouldRun, threshold, bandwidthInMB,
-        parallelThread, version, metrics.getSuccessCount(), metrics.getFailureCount());
+        parallelThread, version, metrics.getSuccessCount(),
+        metrics.getFailureCount(), estimatedTotalSizePendingToMove());
+  }
+
+  private long estimatedTotalSizePendingToMove() {
```

Comment:
1. estimatedTotalSizePendingToMove -> calculateBytesToMove()
2. call calculateBytesToMove from getTasks()
3. add a gobal long field bytesToMove for DiskBalancerService.java to save the result of calculateBytesToMove